visual-inspection/inspection-backend/inspection/model/model.py

```diff
@@ -1,21 +1,70 @@
 import tensorflow as tf
 
 # TODO: Replace with custom model variant
-model = tf.keras.applications.MobileNetV2(input_shape=None,
-                                          include_top=True,
-                                          weights="imagenet")
-
+#model = tf.keras.applications.MobileNetV2(input_shape=None,
+#                                          include_top=True,
+#                                          weights="imagenet")
+model = tf.keras.applications.Xception( include_top=True,
+                                        weights="imagenet",
+                                        classes=1000,
```

Comment:
```suggestion
```

Per [the docs](https://keras.io/api/applications/xception/), this has no effect when `weights` is specified.